doc/api/tools-bilateralize.rst

```diff
@@ -0,0 +1,97 @@
+.. currentmodule:: mesage_ix_models.tools.bilateralize
+Bilateralize (:mod:`.tools.bilateralize`)
+#########################################
+
+.. contents::
+  :local:
+  
+Overview
+========
+This documentation will outline how to bilateralize trade flows in MESSAGEix. By default, trade flows are based on a “global pool” approach, wherein all exporters export energy/commodities to a pool from which importers import based on regionally resolved demands.
```

Comment:
Please see the [documentation code style](https://docs.messageix.org/en/latest/contributing.html#documentation) per "semantic line breaks".

As an intermediate step, start at least every sentence on a new line.